.github/workflows/e2e-test.yml

```
@@ -1,7 +1,9 @@
 name: E2E Test
-run-name: Headless E2E tests for Adyen Magento2 Plugin
+run-name: Headless E2E tests - Basic coverage
```

Names such as `Basic` or `Full` might be ambiguous here IMO. Because, in this test we are running all the E2E tests of the plugin except Express module, which is a separate product and repository. Next comment is also related.

```suggestion
run-name: E2E tests for Adyen Magento2 Plugin
```

---

Makes sense, changed.